### ISSUE TITLE
Modeless dialog blocks the selectors

### ISSUE THREAD

**Issue by jcgueriaud1** [parttio/dramafinder#29]
When you opened a modeless vaadin dialogs in Vaadin 24, the main `div` is aria-hidden. (That means a button in the main page is not clickable from an accessibility point of view)

This means:
`ButtonElement.getByText(page, "Open non-modal 1");`
doesn't work anymore.

You can workaround with this:
```
// the main part is aria-hidden so it's not accessible anymore
        ButtonElement openDialog2Button = new ButtonElement(
                page.getByRole(
                        AriaRole.BUTTON,
                        new Page.GetByRoleOptions().setName("Open non-modal 2")
                                .setIncludeHidden(true)
                ).and(page.locator(ButtonElement.FIELD_TAG_NAME))
        );
```

If  you're trying to use the keyboard outside the dialog you're stuck in the dialog. So in term of usability it seems correct (in short it's not accessible).

Check if Vaadin 25 is changing the behaviour.

<img width="1124" height="901" alt="Image" src="https://github.com/user-attachments/assets/9730efb5-41ad-48af-a555-d71a07401c9f" />

<img width="1964" height="1230" alt="Image" src="https://github.com/user-attachments/assets/9beb5b2a-342c-4e5c-9088-896ca3d6b35c" />

**Comment by jcgueriaud1** [comment]
`.setIncludeHidden(true)` is not needed anymore in Vaadin 25
